src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
Oh, and thanks - that was a useful explanation; crypto is interesting to me, but most of the time I've spent thinking about it is from a consumer perspective so this work (XML crypto) has been pretty illuminating.